modules/ROOT/pages/waf-policy.adoc

```diff
@@ -0,0 +1,8 @@
+= Web Application Security Policy
+
+The Web Application Firewall (WAF) security policy is available for Runtime Fabric inbound traffic to provide protection at the Web application level. WAF policies provide the xref:https://www.owasp.org/index.php/Category:OWASP_ModSecurity_Core_Rule_Set_Project[OWASP Core Rule Set (CRS)] functionality checking for common web application attacks in request and response traffic, which provides an industry standard security baseline. Policies are grouped into the major threat categories for requests and responses.
+
+[NOTE]
+Only the official OWASP CRS rules are supported.
+
+WAF policies are fully integrated with the existing Anypoint Security policy DoS (Denial of Service). When the WAF policy detects errors, it thresholds the errors to the DoS, which can be optionally configured to take actions such as shaping or blocking traffic for a potentially malicious source IP address.
```

Comment:
let's not use threshold as a verb?
malicious-source IP address (since malicious modifies source)?